tencentcloud/resource_tc_kubernetes_cluster.go

```diff
@@ -231,6 +231,27 @@ func TkeCvmCreateInfo() map[string]*schema.Schema {
 				return
 			},
 		},
+		// payment
+		"instance_charge_type": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			ForceNew:     true,
+			Default:      CVM_CHARGE_TYPE_POSTPAID,
+			ValidateFunc: validateAllowedStringValue(TKE_WORKER_CHARGE_TYPE),
+			Description:  "The charge type of instance. Valid values are `PREPAID` and `POSTPAID_BY_HOUR`, The default is `POSTPAID_BY_HOUR`.",
+		},
+		"instance_charge_type_prepaid_period": {
+			Type:         schema.TypeInt,
+			Optional:     true,
+			ValidateFunc: validateAllowedIntValue(CVM_PREPAID_PERIOD),
+			Description:  "The tenancy (time unit is month) of the prepaid instance, NOTE: it only works when instance_charge_type is set to `PREPAID`. Valid values are 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 24, 36.",
+		},
+		"instance_charge_type_prepaid_renew_flag": {
+			Type:         schema.TypeString,
+			Optional:     true,
```

Comment:
should ForceNew and has a defautl value here too
.

Reply:
Will fix, and please check the same code in `resource_tc_instance`.